charts/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -76,6 +76,10 @@ spec:
 {{- if not (eq (int .Values.master.runAsUser) 0) }}
         fsGroup: {{ .Values.master.fsGroup }}
 {{- end }}
+{{- if .Values.master.securityContextCapabilities }}
+        capabilities:
+{{ toYaml .Values.master.securityContextCapabilities | indent 10 }}
+{{- end }}
```

Comment:
```suggestion
         {{- if .Values.master.securityContextCapabilities }}
        capabilities:
          {{ toYaml .Values.master.securityContextCapabilities | nindent 10 }}
        {{- end }}
```

Reply:
I agree with your suggestion about the indentation, but the lines above it aren't indented so it'll look strange unless I reindented the whole file. And i don't want to be that random dude on the internet that steps in and reindents other peoples code :-D

Reply:
You could do it I will take care of indenting the rest sooner or later. Just want to have test cases first to verify that nothing breaks.

Reply:
I've altered the local indentation to the best of my ability